android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetActivity.java

```diff
@@ -87,6 +94,28 @@ public static void launch(Context context, String url) {
         launch(context, options);
     }
 
+    public static void addTopBottomInsets(@NonNull Window w, @NonNull View v) {
+        if (android.os.Build.VERSION.SDK_INT < android.os.Build.VERSION_CODES.VANILLA_ICE_CREAM) return;
+
+        View decorView = w.getDecorView();
+
+        decorView.post(() -> {
+            WindowInsetsCompat insets = ViewCompat.getRootWindowInsets(decorView);
+            if (insets != null) {
+                ViewGroup.MarginLayoutParams params = (ViewGroup.MarginLayoutParams) v.getLayoutParams();
+                params.topMargin = insets.getInsets(WindowInsetsCompat.Type.systemBars()).top;
+                params.bottomMargin = insets.getInsets(WindowInsetsCompat.Type.systemBars()).bottom;
+                v.setLayoutParams(params);
+
+                decorView.setOnApplyWindowInsetsListener((view, windowInsets) -> {
+                    view.setBackgroundColor(Color.BLACK);
```

Comment:
Please use the same background color as the view:

```
public class JitsiMeetView extends FrameLayout {

    /**
     * Background color. Should match the background color set in JS.
     */
    private static final int BACKGROUND_COLOR = 0xFF040404;
```

Probably mmmake it public.